03_launch_mgmt_cluster.sh

```diff
@@ -93,10 +128,16 @@ function update_images(){
     # Strip the tag for image replacement
     OLD_IMAGE="${!OLD_IMAGE_VAR%:*}"
     #shellcheck disable=SC2086
-    kustomize edit set image $OLD_IMAGE=$LOCAL_IMAGE
+    if [ -n "${CAPM3_LOCAL_IMAGE}" ]; then
+      patch_clusterctl
+    else
+      kustomize edit set image $OLD_IMAGE=$LOCAL_IMAGE
+    fi
```

Comment:
this would rather depend on the version, no ?

Reply:
Changed these lines completely due to to other changes